### PR TITLE
Update History.txt and README for release

### PR DESCRIPTION
**Pull request by aggarw13**
Add information of **MQTT Agent** (library + demo), **OTA** (library + demos) and **Device Defender Demo** changes in **History.txt**. Also, mention about the `coreMQTT-Agent-Demos` repository in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,8 @@ Most FreeRTOS Lab libraries can be found in the [FreeRTOS-Labs repository](https
 
 A number of FreeRTOS Lab Demos can be found in the [FreeRTOS Github Organization](https://github.com/FreeRTOS) by searching for "Lab" or following [this link](https://github.com/FreeRTOS?q=Lab&type=&language=) to the search results.
```
Comment (LienNguyen2910) [inline README.md:56]: There is no Lab Demos anymore. Please make clear of basic demo with coreMQTT Agent is in this release, and other demos with coreMQTT Agent can be found in coreMQTT-Agent_Demos repo.

File: History.txt [hunk 2]
```diff
@@ -1,11 +1,18 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
-Changes after FreeRTOS 202012.00 release
+Changes after FreeRTOS 202012.00-LTS and 202012.01-LTS releases
```
Comment (LienNguyen2910) [inline History.txt:3]: Do you mean changes from FreeRTOS 202012.00-LTS to FreeRTOS 202012.01-LTS (this release)?

File: History.txt [hunk 3]
```diff
@@ -1,11 +1,18 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
-Changes after FreeRTOS 202012.00 release
+Changes after FreeRTOS 202012.00-LTS and 202012.01-LTS releases
 
+		+ Update AWS IoT Device Defender and AWS IoT Jobs libraries
+		  to LTS releases.
+		+ Added new OTA library.
```
Comment (LienNguyen2910) [inline History.txt:7]: Need to reword to make sure this new OTA library is v3.0.0, same as the LTS version released in FreeRTOS-LTS 202012.01. Same to Device Defender and Jobs.

File: History.txt [hunk 4]
```diff
@@ -1,11 +1,18 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
-Changes after FreeRTOS 202012.00 release
+Changes after FreeRTOS 202012.00-LTS and 202012.01-LTS releases
 
+		+ Update AWS IoT Device Defender and AWS IoT Jobs libraries
+		  to LTS releases.
+		+ Added new OTA library.
         + Added new coreMQTT Agent library.
+			- Refer to https://github.com/FreeRTOS/coreMQTT-Agent-Demos
```
Comment (LienNguyen2910) [inline History.txt:9]: we added basic coreMQTT Agent demo into P3. Other demos are in coreMQTT-Agent_Demos

File: History.txt [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
+		+ Update AWS IoT Device Defender and AWS IoT Jobs libraries
+		  to LTS releases.
+		+ Added new OTA library.
         + Added new coreMQTT Agent library.
+			- Refer to https://github.com/FreeRTOS/coreMQTT-Agent-Demos 
+			  for demo examples connecting to AWS
         + Demo Updates
-                - MQTT Agent
-
+                - Refactored MQTT Multitasking demo to use MQTT Agent
```
Comment (LienNguyen2910) [inline History.txt:12]: Add coreMQTT Agent demo

**Review by LienNguyen2910 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aggarw13) [reply README.md:56]: There do exist a number of Labs repositories which are still relevant to point out
https://github.com/FreeRTOS?q=Lab&type=&language=&sort=

File: History.txt (hunk 2 above)
Comment (aggarw13) [reply History.txt]: Fixed

File: History.txt (hunk 5 above)
Comment (aggarw13) [reply History.txt]: Replaced

File: History.txt [hunk 6]
```diff
@@ -1,18 +1,21 @@
...
-		+ Added new OTA library.
+		+ Released LTS versions of AWS IoT Device Defender, AWS IoT Jobs and
+		  AWS IoT Over-the-air-Updates (OTA) libraries
+		  	- AWS IoT Device Defender v1.1.0
+		  	- AWS IoT Jobs v1.1.0
+			- AWS IoT Over-the-air-Updates (OTA) v3.0.0
         + Added new coreMQTT Agent library.
-			- Refer to https://github.com/FreeRTOS/coreMQTT-Agent-Demos 
-			  for demo examples connecting to AWS
+			- For examples of using coreMQTT Agent library to communicate with
```
Comment (LienNguyen2910) [inline History.txt:11]: Please move this under Demo Updates/Add coreMQTT Agent demo

File: History.txt (hunk 3 above)
Comment (aggarw13) [reply History.txt]: I have explicitly mentioned the LTS release versions of the libraries

File: README.md (hunk 1 above)
Comment (LienNguyen2910) [reply README.md:56]: OK, i confused with Lab-Project-coreMQTT.

**Review by LienNguyen2910 — CHANGES_REQUESTED** [review]

File: History.txt (hunk 6 above)
Comment (aggarw13) [reply History.txt]: Done

**Review by LienNguyen2910 — APPROVED** [review]

**Review by gkwicker — APPROVED** [review]

**Pull request merged**